templates/nro/.circleci/config.yml.tmpl

```diff
@@ -312,6 +324,13 @@ jobs:
       <<: *common_environment
       <<: *release_environment
     <<: *finish_staging_steps
+  
+  rollback-staging:
```

Comment:
Not sure if we should rename either, but this has the same name as the approval job in the `release` workflow.

Reply:
Didn't notice any conflict during testing, since it's in a different workflow.